bot_modules/irc_kudos.js

```diff
@@ -4,13 +4,17 @@ var what_max_length = 32;
 
 var process = function(words, operator) {
 	// build what
-	what = words.join(' ');
+	var what = words.join(' ');
```

Comment:
why not `let` ?

Reply:
the point of the rewind was to undo most of the es6 features lol

Reply:
i think `let` here would be good though; i thought the point of the commit was just to remove unnecessary aesthetic changes

Reply:
you're probably right, but the main point of this patch is to pile the few changes I've had since the ES6 aesthetic changes back on top of after the repo is rewinded

Reply:
I just use var always because I'm a sloppy burly asshole

Reply:
🍆